docs/documentation/quartz-3.x/how-tos/multiple-triggers.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Multiple Triggers
+---
+
+One nice thing about Quartz is that you can split the job registration from the 
+triggers. One nice use case is being able to split out the JobData for each
+trigger. 
+
+Below, we have two triggers, each with their own set of data, but
+we only had to register the one job.
+
+
+```csharp
+public Task DoSomething(IScheduler schedule, CancellationToken ct)
+{
+    var job = JobBuilder.Create<AnExampleJob>()
```

Comment:
In MassTransit we have a common idiom of trying to use "orders" and "order placement" all the time. I find it helps keep the documentation and the content in a cohesive direction. Do you have a simple domain that would work well for scheduling?

Something that we can do at both a simple level for the intro docs, but has some good complexity for the harder / richer features?

Reply:
[MassTransit also has the scheduler-backed feature of scheduled send](https://masstransit-project.com/advanced/scheduling/) which I like, but you probably are quite familiar with this library 🙂 

I might have used Quartz in limited domain as there's been people complaining about couple millisecond imprecision for some near real-time processing.. But I think many have been using for things like:

* subscriptions (like daily digest), either user-configured or registering to predefined "triggers" like "at most daily", "at most weekly"
* synchronizing data between systems when there's no real activity/trigger/observer communication - load/sync changed content from CMS daily, sync user preferences nightly etc
* cleanup / maintenance - [OpenIddict cleans some expired data for example](https://kevinchalet.com/2020/10/02/introducing-quartz-net-support-and-new-languages-in-openiddict-3-0-beta4/)) , [Bitwarden uses in multiple ways](https://github.com/bitwarden/server/search?q=quartz)
* generate/prepare reports for past day nightly that would otherwise be expensive to do real-time

Not sure what would be easy one to extend as these are usually highly use case specific and great in that way that they follow single responsibility principle. Following the MS weather forecast service example it could be as simple as getting forecast data and persisting hourly - basically the in-memory caching version without the in-memory caching 🤷🏻‍♂️